assistive_gym/envs/human_comfort_env.py

```diff
@@ -70,6 +74,86 @@ def _get_obs(self, agent=None): # not needed
     def get_total_force(self):
         pass
 
+    
+    def get_reba_score(self):
```

Comment:
we might want to take this one and add to the human_urdf as a method under human object. It is optional, we could do that later once everything working